src/meta/model/src/exactly_once_iceberg_sink.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2025 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sea_orm::entity::prelude::*;
+use sea_orm::{DeriveEntityModel, DeriveRelation, EnumIter};
+
+use crate::{Epoch, SinkId};
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Default)]
+#[sea_orm(table_name = "exactly_once_iceberg_sink_metadata")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub sink_id: SinkId,
+    #[sea_orm(primary_key, auto_increment = false)]
```

Comment:
I think we can't define 2 primary keys here. cc @yezizp2012 Do you know how to define an index on a column with sea_orm?

Reply:
Well, I think you can create the table via migration locally and then run the following command to generate the model files:

`sea-orm-cli generate entity`

Check more details here: src/meta/model/src/README.md

Reply:
It appears that defining two primary keys for a model is feasible. An example was given here: https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure/#composite-key

Reply:
Yep. Thanks. We have checked the table definition from the meta store and it works.